lasagne/layers/conv.py

```diff
@@ -1047,88 +1352,47 @@ class TransposedConv3DLayer(BaseConvLayer):  # pragma: no cover
 
     Notes
     -----
-    The transposed convolution is implemented as the backward pass of a
-    corresponding non-transposed convolution. It can be thought of as dilating
-    the input (by adding ``stride - 1`` zeros between adjacent input elements),
-    padding it with ``filter_size - 1 - crop`` zeros, and cross-correlating it
-    with the filters. See [1]_ for more background.
+    See `BaseTransposedConvLayer` for more.
 
     Examples
     --------
     To transpose an existing convolution, with tied filter weights:
 
     >>> from lasagne.layers import Conv3DLayer, TransposedConv3DLayer
-    >>> conv = Conv3DLayer((None, 1, 32, 32, 32), 16, 3, stride=2, pad=2)
+    >>> conv = Conv3DLayer((None, 1, 32, 64, 64), 16, 3, stride=2, pad=2)
     >>> deconv = TransposedConv3DLayer(conv, conv.input_shape[1],
     ...         conv.filter_size, stride=conv.stride, crop=conv.pad,
     ...         W=conv.W, flip_filters=not conv.flip_filters)
-
-    References
-    ----------
-    .. [1] Vincent Dumoulin, Francesco Visin (2016):
-           A guide to convolution arithmetic for deep learning. arXiv.
-           http://arxiv.org/abs/1603.07285,
-           https://github.com/vdumoulin/conv_arithmetic
     """
-    def __init__(self, incoming, num_filters, filter_size,
-                 stride=(1, 1, 1), crop=0, untie_biases=False,
+    def __init__(self, incoming, num_filters, filter_size, stride=1,
+                 crop=0, untie_biases=False,
```

Comment:
I'd prefer `(1, 1, 1)` just for consistency with the forward convolutional layers.